manifest.json

```
@@ -0,0 +1,23 @@
+{
+  "templates": [
+    {
+      "id": "connectors",
+      "display_name": "Start with REST",
```

Where does the `display_name` display?

Context: I'd love to leave the `path`s sparse, but maybe we can make the `display_name` more descriptive? 

```suggestion
      "display_name": "Start with existing REST APIs",
```

---

> can we think of a way that communicates that with one you are using pre-existing (REST) APIs and the other you are building a new API using TypeScript?

This is a good point! If we're open to another level of nesting, what about something like:

```
|-- 📁 start-with-rest
|-- 📁 start-with-recommended-libraries (or `start-with-graphql-libraries` to be more specific?)
      |-- 📁 typescript-apollo-server
```

We could also consider something like `📁 start-with-resolvers` instead—our concern with that was that it might confuse folks if they're unfamiliar with the term, but that _could_ also be an intentional decision we make to help guide them toward the more familiar one (cc: @Meschreiber)

---

`display_name` is meant to be shown in the selection prompt, similar to what we do in `rover template`. The path field is currently only used by init to locate the template files in the repo.

That said, if we think the manifest needs more fields (or a different structure) to support a better experience, now’s the perfect time to make that change. I’m happy to adjust it. Also, if having the manifest in the root feels misleading for folks landing on the repo via search, we could consider moving it to a subfolder to keep things cleaner.